README.md

```diff
@@ -1920,6 +1926,174 @@ map.addEntryListener(mapEventListener).then(function () {
 });
 ```
 
+#### 7.5.2.2. Listening for MultiMap Events
+
+In MultiMap, you can listen to map-wide or entry-based events by using the functions in the `EntryListener` interface. Every function type in this interface is one of the `EntryEventListener` and `MapEventListener` types. To listen to these events, you need to implement the relevant `EntryEventListener` and `MapEventListener` functions in the `EntryListener` interface. 
+
+An entry-based event is fired after the operations that affect a specific entry. For example, `MultiMap.put()`, `MultiMap.remove()`. You should use the `EntryEventListener` type to listen to these events. An `EntryEvent` object is passed to the listener function.
+
+See the following example.
+
+```javascript
+var entryEventListener = {
+    added: function (entryEvent) {
+        console.log('Entry Added:', entryEvent.key, entryEvent.value); // Entry Added: 1 Furkan
+    }
+};
+return mmp.addEntryListener(entryEventListener, undefined, true).then(function () {
```

Comment:
I think we should explain what the second and third arguments do since they are in the example.